git/git_remotes.md

```diff
@@ -1,7 +1,7 @@
 # Git Remotes
-In this lesson, we will cover the `git remote` command and remote repositories. 
+In this lesson, we will cover the `git remote` command and remote repositories.
 
-A remote tells Git where the other versions of a project are stored. These versions are housed in "remote repositories," usually on another person's computer or a server.
+A remote on Git tracks the different versions of a project. The project versions are stored on "remote repositories" such as another person's computer or a server.
```

Comment:
This change is incorrect - a remote is a pointer to a specific repository. A remote does not "track" versions